Update the Upgrading Using a Direct Download guide section. (#13684)
elastic/logstash: docs/static/upgrading.asciidoc

This procedure downloads the relevant Logstash binaries directly from Elastic.

. Shut down your Logstash pipeline, including any inputs that send events to Logstash.
. Download the https://www.elastic.co/downloads/logstash[Logstash installation file] that matches your host environment.
. Backup your `config/` and `data/` folders in a temporary space.
. Delete your Logstash directory.
. Unpack the installation file into the folder that contained the Logstash directory that you just deleted.
. Restore the `config/` and `data/` folders that were previously saved, overwriting the folders created during the unpack operation.
. Test your configuration file with the `logstash --config.test_and_exit -f <configuration-file>` command.
Configuration options for

> some Logstash plugins have changed in the 7.x release.

Reviewer: ```suggestion
some Logstash plugins have changed.
```

Reviewer: What do you think about this suggestion, @andsel?  It seems like some plugins changed in 7.x and others changed between 7.x and 8.0.

Author: correct, we can drop that part of the sentence.

> 1. Shut down your Logstash pipeline, including any inputs that send events to Logstash.
> 2. Download the https://www.elastic.co/downloads/logstash[Logstash installation file] that matches your host environment.

Reviewer: ```suggestion
. Shut down your Logstash pipeline, including any inputs that send events to Logstash.
. Download the https://www.elastic.co/downloads/logstash[Logstash installation file] that matches your host environment.
```

Reviewer: This notation sets up autonumbering. :-)

> 3. Backup your `config/` and `data/` folders in a temporary space.
> 4. Remove your Logstash directory.
> 5. Unpack the installation file into the same folder your existing Logstash directory was present.
> 6. Restore the `config/` and `data/` folders that was previously saved, overwriting the folders created during the unpack operation.
> 7. Test your configuration file with the `logstash --config.test_and_exit -f <configuration-file>` command.

Reviewer: ```suggestion
. Backup your `config/` and `data/` folders in a temporary space.
. Delete your Logstash directory.
. Unpack the installation file into the folder that contained the Logstash directory that you just deleted.
. Restore the `config/` and `data/` folders that were previously saved, overwriting the folders created during the unpack operation.
. Test your configuration file with the `logstash --config.test_and_exit -f <configuration-file>` command.
```

Configuration options for
some Logstash plugins have changed in the 7.x release.

> 8. Restart your Logstash pipeline after updating your configuration file.

Reviewer: ```suggestion
. Restart your Logstash pipeline after updating your configuration file.
```
